### ISSUE TITLE
"the `asm!` macro is not allowed in naked functions" in recent nightlies

### ISSUE THREAD

**Issue by VisualEhrmanntraut** [nbdd0121/unwinding#38]
```
error[E0787]: the `asm!` macro is not allowed in naked functions
  --> /Users/visual/.cargo/git/checkouts/unwinding-943b7a48f658f090/76a9f9d/src/unwinder/arch/x86_64.rs:64:9
   |
64 | /         asm!(
65 | |             "
66 | |             sub rsp, 0x98
67 | |             .cfi_def_cfa_offset 0xA0
...  |
94 | |             options(noreturn)
95 | |         );
   | |_________^ consider using the `naked_asm!` macro instead

For more information about this error, try `rustc --explain E0787`.
error: could not compile `unwinding` (lib) due to 1 previous error
```

**Comment by sunfishcode** [comment]
There is now a fix for this on the dev branch in e651eadfa6c68fa46559199d435aa35ff9b84529.

**Comment by VisualEhrmanntraut** [comment]
Seems to work.

**Comment by xobs** [comment]
This is causing build failures of libstd on nightly for Xous (see https://github.com/rust-lang/rust/issues/131602). Will a new version be tagged soon?

**Comment by nbdd0121** [comment]
Published 0.2.3
